health-services/project-factory/src/server/utils/genericUtils.ts

```
@@ -928,15 +928,35 @@ async function generateFacilityAndBoundarySheet(tenantId: string, request: any,
+
+  // Clear and replace original array
+  headers.length = 0;
+  headers.push(...newHeaders);
+}
+
 async function generateUserSheet(request: any, localizationMap?: { [key: string]: string }, filteredBoundary?: any, userData?: any, fileUrl?: any) {
   const tenantId = request?.query?.tenantId;
   const type = request?.query?.type || request?.body?.ResourceDetails?.type;
   const typeWithoutWith = type.includes('With') ? type.split('With')[0] : type;
   let schema: any;
   const isUpdate = fileUrl ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)